library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -893,17 +910,18 @@ public void focusOnUserLocation(boolean focusOnMyLocation) {
     @Override
     public void focusOnUserLocation(boolean focusOnMyLocation, Float radius) {
         if (focusOnMyLocation) {
-            isMapScrolled = false;
+            //isMapScrolled = false;
             changeImageButtonResource(currentLocationBtn, R.drawable.ic_cross_hair_blue);
 
             // Enable the listener & show the current user location
             updateUserLocationOnMap = true;
+            updateCameraUserLocationOnMap = true;
```

Comment:
What is the aim of this, it seems not to have an effect when the background service is giving me different location as the one which is supplied by the `GoogleLocationClient`?

Why is this commented?
https://github.com/onaio/kujaku/blob/be6ccc95dcec2d30e83318b1e937c92a30df6905/library/src/main/java/io/ona/kujaku/views/KujakuMapView.java#L760

Reply:
I will delete the comments because the method `focusOnUserLocation `already set the `updateCameraUserLocationOnMap ` variable. 
The aim of this new variable is to separate the 2 differents behaviours that were not isolated before :
- Update the user location on Map
- Update the camera to focus and zoom on the user location on Map

Reply:
Is it being used right now, it seems that both behaviours happen in tandem and both variables holding flags for each always have the same value, either `false` at the same time or `true`?

Reply:
No, you can update the user location without updating the camera on this user location. This behaviour was not possible before

Reply:
OK.